backend/database_handler/models.py

```diff
@@ -117,18 +118,33 @@ class Transactions(Base):
     )
 
 
-class TransactionsAudit(Base):
-    __tablename__ = "transactions_audit"
-    __table_args__ = (PrimaryKeyConstraint("id", name="transactions_audit_pkey"),)
+class RollupTransactions(Base):
+    __tablename__ = "rollup_transactions"
+    __table_args__ = (PrimaryKeyConstraint("id", name="rollup_transactions_pkey"),)
 
     id: Mapped[int] = mapped_column(Integer, primary_key=True, init=False)
```

Comment:
@kstroobants the ID should be the transaction hash

Reply:
Fixed in eca17e5fbf511c790d2025babe11cdf8c3bec6b7. Replaced id by transaction_hash.
It cannot be the transaction_hash from the genlayer because then the table contains multiple entries with the same primary key when adding rollup transactions for each consensus state change and that is not allowed.
For now, I used _generate_transaction_hash() to make a transaction_hash for the rollup_transaction. Maybe it needs to be changed because there is no type.